src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -716,33 +671,21 @@
 helps['vm nic list'] = """
     type: command
     short-summary: List the NICs available on a VM.
-    examples:
-        - name: List all of the NICs on a VM.
-          text: az vm nic list -g MyResourceGroup --vm-name MyVm
 """
 
 helps['vm nic add'] = """
     type: command
     short-summary: Add existing NICs to a VM.
-    examples:
-        - name: Add two NICs to a VM.
-          text: az vm nic add -g MyResourceGroup --vm-name MyVm --nics nic_name1 nic_name2
```

Comment:
Considering how many people don't understand what a "space-separated list" means within the context of the CLI, I would keep this.  Otherwise people try:
`--nics "a b"` (this is a string)
`--nics "a,b"` (this is also a string)

Reply:
The documentation does not even make a reference to "space-separated values". Documentation should be fixed first. Place some trust in developers to also understand that `"x y z"` is a single value under `bash`. The target for this documentation is not somebody new to bash, it is somebody new to the CLI. Our job is not to teach them shell basics.

Reply:
Usually this anti-pattern originates not from bash users but users of the previous Xplat CLI.  However, I'm fine with omitting the example since, as you mentioned, the fact that it is space separated should be called out in the help.